app/controllers/foreman_inventory_upload/uploads_settings_controller.rb

```diff
@@ -0,0 +1,25 @@
+module ForemanInventoryUpload
+  class UploadsSettingsController < ::ApplicationController
+    def index
+      render json: {
+        autoUploadEnabled: Setting[:allow_auto_inventory_upload],
+        hostObfuscationEnabled: Setting[:obfuscate_inventory_hostnames],
+        ipsObfuscationEnabled: Setting[:obfuscate_inventory_ips],
+        excludePackages: Setting[:exclude_installed_packages],
+        CloudConnectorStatus: ForemanInventoryUpload::UploadsSettingsController.cloud_connector_status,
+      }, status: :ok
+    end
+
+    def set_advanced_setting
+      Setting[params.require(:setting)] = ActiveModel::Type::Boolean.new.cast(params.require(:value))
+      index
+    end
+
+    def self.cloud_connector_status
```

Comment:
we will get currently the `CloudConnectorStatus` from two spots, one on the Settings API call and multiple times in the inventory account polling,
for now, let's keep it only in the polling